packages/api/src/resolvers/subscriptions/index.ts

```diff
@@ -47,61 +47,56 @@ export const subscriptionsResolver = authorized<
   SubscriptionsSuccessPartial,
   SubscriptionsError,
   QuerySubscriptionsArgs
->(
-  async (
-    _obj,
-    { sort, type = SubscriptionType.Newsletter }, // default to newsletter
-    { claims: { uid }, log }
-  ) => {
-    log.info('subscriptionsResolver')
+>(async (_obj, { sort, type }, { claims: { uid }, log }) => {
+  log.info('subscriptionsResolver')
 
-    analytics.track({
-      userId: uid,
-      event: 'subscriptions',
-      properties: {
-        env: env.server.apiEnv,
-      },
-    })
+  analytics.track({
+    userId: uid,
+    event: 'subscriptions',
+    properties: {
+      env: env.server.apiEnv,
+    },
+  })
 
-    try {
-      const sortBy =
-        sort?.by === SortBy.UpdatedTime ? 'lastFetchedAt' : 'createdAt'
-      const sortOrder = sort?.order === SortOrder.Ascending ? 'ASC' : 'DESC'
-      const user = await getRepository(User).findOneBy({ id: uid })
-      if (!user) {
-        return {
-          errorCodes: [SubscriptionsErrorCode.Unauthorized],
-        }
+  try {
+    const sortBy =
+      sort?.by === SortBy.UpdatedTime ? 'lastFetchedAt' : 'createdAt'
+    const sortOrder = sort?.order === SortOrder.Ascending ? 'ASC' : 'DESC'
+    const user = await getRepository(User).findOneBy({ id: uid })
+    if (!user) {
+      return {
+        errorCodes: [SubscriptionsErrorCode.Unauthorized],
       }
+    }
 
-      const queryBuilder = getRepository(Subscription)
-        .createQueryBuilder('subscription')
-        .leftJoinAndSelect('subscription.newsletterEmail', 'newsletterEmail')
-        .where({
-          user: { id: uid },
-          type,
-        })
-
-      // only return active subscriptions for newsletter
-      if (type === SubscriptionType.Newsletter) {
-        queryBuilder.andWhere({ status: SubscriptionStatus.Active })
-      }
+    const queryBuilder = getRepository(Subscription)
+      .createQueryBuilder('subscription')
+      .leftJoinAndSelect('subscription.newsletterEmail', 'newsletterEmail')
+      .where({
+        user: { id: uid },
+        status: SubscriptionStatus.Active,
```

Comment:
We probably need add `status` to the request payload because we need to show `Paused` subscriptions on RSS page

So for newsletters, we can get subscriptions which `status = active` and for rss feeds, we don't put `status` in the query

Reply:
Ah ok, that makes it a little tricky, so i guess right now for feeds if its paused it sets it to UNSUBSCRIBED

Reply:
Makes sense, I will add some tests for these cases.